tests/test_EN_US.py

```diff
@@ -38,7 +38,10 @@ def test_en_us(self):
             "nine point oh nine",
             "point eight eight",
             "point oh zero five",
-            "a thousand and fifty four")
+            "a thousand and fifty four",
+            "three sixty",
+            "seven eleven",
```

Comment:
Why seven eleven in particular? The correct form is usually `7-Eleven`, so I don't think this is the best test.

Reply:
fair point. i just moved it up from the negatives test but i'll change it to something more appropriate.